cmake.config.h.in

```diff
@@ -0,0 +1,416 @@
+#cmakedefine HAVE_STDINT_H 1
+
+/* Define to 1 if you have the <stdlib.h> header file. */
+#cmakedefine HAVE_STDLIB_H 1
+
+/* Define to 1 if you have the <strings.h> header file. */
+#cmakedefine HAVE_STRINGS_H 1
+
+/* Define to 1 if you have the <string.h> header file. */
+#cmakedefine HAVE_STRING_H 1
+
+/* Define to 1 if you have the `sysctl' function. */
+#define HAVE_SYSCTL 1
+
+/* Define to 1 if you have the <sys/stat.h> header file. */
+#cmakedefine HAVE_SYS_STAT_H 1
+
+/* Define to 1 if you have the <sys/sysctl.h> header file. */
+#cmakedefine HAVE_SYS_SYSCTL_H 1
+
+/* Define to 1 if you have the <sys/time.h> header file. */
+#cmakedefine HAVE_SYS_TIME_H 1
+
+/* Define to 1 if you have the <sys/types.h> header file. */
+#cmakedefine HAVE_SYS_TYPES_H 1
+
+/* Define to 1 if you have the `tanl' function. */
+/* #undef HAVE_TANL */
+
+/* Define if we have a threads library. */
+#cmakedefine HAVE_THREADS 1
+
+/* Define to 1 if you have the `time_base_to_time' function. */
+/* #undef HAVE_TIME_BASE_TO_TIME */
+
+/* Define to 1 if the system has the type `uintptr_t'. */
+#define HAVE_UINTPTR_T 1
+
+/* Define to 1 if you have the <unistd.h> header file. */
+#cmakedefine HAVE_UNISTD_H 1
+
+/* Define to 1 if you have the `vprintf' function. */
+#define HAVE_VPRINTF 1
+
+/* Define to enable IBM VSX optimizations. */
+/* #undef HAVE_VSX */
+
+/* Define to 1 if you have the `_mm_free' function. */
+/* #undef HAVE__MM_FREE */
+
+/* Define to 1 if you have the `_mm_malloc' function. */
+/* #undef HAVE__MM_MALLOC */
+
+/* Define if you have the UNICOS _rtc() intrinsic. */
+/* #undef HAVE__RTC */
+
+/* Define to the sub-directory in which libtool stores uninstalled libraries.
+   */
+#define LT_OBJDIR ".libs/"
+
+/* Define to 1 if your C compiler doesn't accept -c and -o together. */
+/* #undef NO_MINUS_C_MINUS_O */
+
+/* Name of package */
+#define PACKAGE "fftw"
+
+/* Define to the address where bug reports for this package should be sent. */
+#define PACKAGE_BUGREPORT "fftw@fftw.org"
+
+/* Define to the full name of this package. */
+#define PACKAGE_NAME "fftw"
+
+/* Define to the full name and version of this package. */
+#define PACKAGE_STRING "fftw @FFTW_VERSION@"
+
+/* Define to the one symbol short name of this package. */
+#define PACKAGE_TARNAME "fftw"
+
+/* Define to the home page for this package. */
+#define PACKAGE_URL ""
+
+/* Define to the version of this package. */
+#define PACKAGE_VERSION "@FFTW_VERSION@"
+
+/* Define to necessary symbol if this constant uses a non-standard name on
+   your system. */
+/* #undef PTHREAD_CREATE_JOINABLE */
+
+/* The size of `double', as computed by sizeof. */
+#define SIZEOF_DOUBLE 8
+
+/* The size of `fftw_r2r_kind', as computed by sizeof. */
+#define SIZEOF_FFTW_R2R_KIND 4
+
+/* The size of `float', as computed by sizeof. */
+#define SIZEOF_FLOAT 4
+
+/* The size of `int', as computed by sizeof. */
+#define SIZEOF_INT 4
+
+/* The size of `long', as computed by sizeof. */
+#define SIZEOF_LONG 8
```

Comment:
This is incorrect for Windows, for example.  Presumably these should be determined by [CheckTypeSize](https://cmake.org/cmake/help/v3.0/module/CheckTypeSize.html) in cmake.